core-api/src/main/java/org/silverpeas/core/util/DateUtil.java

```diff
@@ -70,9 +73,8 @@
   public static final FastDateFormat ICALDATE_FORMATTER;
   public static final FastDateFormat ICALUTCDATE_FORMATTER;
   public static final FastDateFormat ISO8601_FORMATTER;
-  private static final DateTimeFormatter CUSTOM_FORMATTER =
-      DateTimeFormatter.ofPattern("yyyy/MM/dd");
-  public static final DateTimeFormatter LUCENE_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMdd");
+  private static final DateTimeFormatter CUSTOM_FORMATTER = ofPattern("yyyy/MM/dd");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "yyyy/MM/dd" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)